### PR TITLE
Fixes Microservices ebook content

### PR DESCRIPTION
**Pull request by sughosneo**
This PR updates following content in microservices ebook. 

1. Updates docker terminology 
2. Changes aspnet docker image tags
3. Fixes container orchestrator options for Visual Studio docker tools

### CODE REVIEW

File: docs/architecture/microservices/container-docker-introduction/docker-terminology.md [hunk 1]
```diff
@@ -23,29 +23,27 @@ docker build
...
-
-**Docker Enterprise Edition (EE)**: An enterprise-scale version of Docker tools for Linux and Windows development.
+**Docker Desktop**: Development tools for Windows and macOS for building, running, and testing containers locally. Docker Desktop for Windows provides development environments for both Linux and Windows Containers. The Linux Docker host on Windows is based on a [Hyper-V](https://www.microsoft.com/cloud-platform/server-virtualization) virtual machine. The host for Windows Containers is directly based on Windows. Docker Desktop for Mac is based on the Apple Hypervisor framework and the [xhyve hypervisor](https://github.com/mist64/xhyve), which provides a Linux Docker host virtual machine on Mac OS X. Docker Desktop for Windows and for Mac replaces Docker Toolbox, which was based on Oracle VirtualBox.
```
Comment (nishanil) [inline docs/architecture/microservices/container-docker-introduction/docker-terminology.md:40]:
```suggestion
**Docker Desktop**: Development tools for Windows and macOS for building, running, and testing containers locally. Docker Desktop for Windows provides development environments for both Linux and Windows Containers. The Linux Docker host on Windows is based on a [Hyper-V](https://www.microsoft.com/cloud-platform/server-virtualization) virtual machine. The host for Windows Containers is directly based on Windows. Docker Desktop for Mac is based on the Apple Hypervisor framework and the [xhyve hypervisor](https://github.com/mist64/xhyve), which provides a Linux Docker host virtual machine on macOS. Docker Desktop for Windows and for Mac replaces Docker Toolbox, which was based on Oracle VirtualBox.
```

**Comment by nishanil** [comment]
@sughosneo Thanks for fixing the docker versions. Left a minor comment to address.

**Review by IEvangelist — COMMENTED** [review]
Looks great, we should be able to approve / merge this once you've accepted the suggestions.

**Review by IEvangelist — APPROVED** [review]
Approved, pending acceptance of suggestions.

**Pull request merged**